easybuild/easyblocks/o/openfoam.py

```diff
@@ -489,9 +490,10 @@ def sanity_check_step(self):
             openfoamdir_path = os.path.join(self.installdir, self.openfoamdir)
             motorbike_path = os.path.join(openfoamdir_path, 'tutorials', 'incompressible', 'simpleFoam', 'motorBike')
             if os.path.exists(motorbike_path):
+                test_dir = tempfile.mkdtemp()
```

Comment:
This may cause issues if the example is run on more than one node

Reply:
but it is fixable by setting the `--tmpdir` option to be on a parallel file system

Reply:
I think it's safe to assume that in general EasyBuild isn't running in a multi-node context...